commands/src/main/java/org/wildfly/extras/creaper/commands/orb/ChangeOrb.java

```diff
@@ -0,0 +1,606 @@
+package org.wildfly.extras.creaper.commands.orb;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import org.jboss.dmr.ModelNode;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.ServerVersion;
+import org.wildfly.extras.creaper.core.offline.OfflineCommand;
+import org.wildfly.extras.creaper.core.offline.OfflineCommandContext;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Batch;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+/**
+ * Creaper command which allows settings of orb subsystem
+ *
+ * <ul>
+ *   <li>jacorb for EAP 6.x (<= WFLY 8)
+ *   <li>openjdk iiop for EAP 7.x (>= WFLY 9)
```

Comment:
I'd prefer stating the upstream relationship first. If you want to add EAP relationship, that's fine, but _that_ belongs to the parens. Like:

```java
<li>JacORB for AS7 and WildFly <= 8 (EAP 6)</li>
<li>OpenJDK ORB for WildFly >= 9 (EAP 7)</li>
```

Personally, I don't feel the EAP bits are needed, but feel free to keep them.